doc/classes/ProjectSettings.xml

```diff
@@ -520,6 +520,9 @@
 		<member name="debug/gdscript/warnings/onready_with_export" type="int" setter="" getter="" default="2">
 			When set to [code]warn[/code] or [code]error[/code], produces a warning or an error respectively when the [code]@onready[/code] annotation is used together with the [code]@export[/code] annotation, since it may not behave as expected.
 		</member>
+		<member name="debug/gdscript/warnings/array_copied_in_argument" type="int" setter="" getter="" default="1">
+			When set to [code]warn[/code] or [code]error[/code], produces a warning or an error respectively when a [code]Packed*Array[/code] is passed as an argument with type [code]Array[/code], since the array will be copied and converted rather than passed by reference.
```

Comment:
```suggestion
			When set to [code]warn[/code] or [code]error[/code], produces a warning or an error respectively when a [code]Packed*Array[/code] is passed as an argument of type [Array], since the array will be copied and converted rather than passed by reference.
```